cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -0,0 +1,535 @@
+
+      - !variable
+        id: issuer
+
+      - !group hosts
+
+      - !permit
+        role: !group hosts
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/keycloak/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/keycloak/hosts
+      member: !host myapp
+
+    - !host
+      id: alice
+      annotations:
+        authn-jwt/keycloak/email: alice@conjur.net
+
+    - !grant
+      role: !group conjur/authn-jwt/keycloak/hosts
+      member: !host alice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "provider-uri" variable with OIDC value from env var "PROVIDER_URI"
+    And I successfully set authn-jwt "token-app-property" variable with OIDC value from env var "ID_TOKEN_USER_PROPERTY"
+    And I successfully set authn-jwt "issuer" variable with OIDC value from env var "PROVIDER_ISSUER"
+    And I fetch an ID Token for username "alice" and password "alice"
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the ID token
+    Then host "alice" has been authorized by Conjur
+    And The following lines appear in the log after my savepoint:
+      |                                                                                                 |
+      |CONJ00076I Selected signing key interface: 'provider-uri'                                        |
+      |CONJ00072I Fetching JWKS from 'https://keycloak:8443/auth/realms/master'...                      |
+      |CONJ00054I "issuer" value will be taken from 'cucumber:variable:conjur/authn-jwt/keycloak/issuer'|
+      |CONJ00055I Retrieved "issuer" with value 'http://keycloak:8080/auth/realms/master'               |
+      |CONJ00098I Successfully found JWT identity 'host/alice'                                          |
+    And I update the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !delete
+        record: !variable provider-uri
+    """
+    And I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !variable
+        id: jwks-uri
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" in service "keycloak"
+    And I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable value to "host" in service "keycloak"
+    And I permit host "myapp" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt using given keycloak service ID and without account in url
+    Then host "myapp" has been authorized by Conjur
```

Comment:
This is good place for the then to be . Changed all what above until the when to And